bp_common/syn/vivado/xdc/design.xdc

```diff
@@ -0,0 +1,2 @@
+create_clock -period 50 [get_ports clk_i]
```

Comment:
nit, but just do 30 to keep runtime low.  Without retiming, the frequency is kind of meaningless anyway